transports/transport-highway/src/main/java/io/servicecomb/transport/highway/HighwayClient.java

```diff
@@ -62,7 +66,13 @@ public void init(Vertx vertx) throws Exception {
 
   private TcpClientConfig createTcpClientConfig() {
     TcpClientConfig tcpClientConfig = new TcpClientConfig();
-    tcpClientConfig.setRequestTimeoutMillis(AbstractTransport.getRequestTimeout());
+    DynamicLongProperty prop = AbstractTransport.getRequestTimeoutProperty();
+    prop.addCallback(new Runnable(){
```

Comment:
It's better to add a comment on this line to support setup the request timeout dynamically.